apps/yarn.lock

```diff
@@ -60,7 +60,7 @@
     temporal latest
   optionalDependencies:
     browser-serialport latest
-    firmata "^0.12.0"
+    firmata latest
```

Comment:
I'm slightly confused; if the versions of `playground-io` and `johnny-five` that we're depending on now have overlapping `firmata` requirements, why is it also necessary to require `firmata` explicitly ourselves in order to have them both resolve to the same version? Is it simply not the case that yarn will try to reduce version overlap for dependencies of our dependencies, and that it will only both to check against duplicates of top-level dependencies?

Naturally we'll still want firmata as a top-level dependency since we're referencing it, but it makes me unhappy that that is necessary to prevent duplicate versions.